projects/canopy/src/lib/filter-buttons/filter-button/filter-button.component.scss

```diff
@@ -0,0 +1,99 @@
+@import '../../../styles/mixins';
+
+.lg-filter-button__input {
+  @include lg-visually-hidden;
+}
+
+.lg-filter-button__label {
+  display: inline-block;
+  max-height: var(--space-lg);
+  border: solid var(--border-width) var(--filter-button-bg-color-active);
```

Comment:
We usually use `btn` instead of `button` in our variables ;)

Reply:
Good to know. Done.